CUDADataFormats/SiPixelDigi/interface/SiPixelDigisCUDA.h

```diff
@@ -6,9 +6,11 @@
 #include "HeterogeneousCore/CUDAUtilities/interface/device_unique_ptr.h"
 #include "HeterogeneousCore/CUDAUtilities/interface/host_unique_ptr.h"
 #include "HeterogeneousCore/CUDAUtilities/interface/cudaCompat.h"
+#include "CUDADataFormats/SiPixelDigi/interface/SiPixelDigisCUDASOAView.h"
 
 class SiPixelDigisCUDA {
 public:
+  using m_store_type = uint16_t;
```

Comment:
please use the naming convention for types rather than data members:
```suggestion
  using StoreType = uint16_t;
```